testing/common/multinode_setup.go

```diff
@@ -114,6 +116,10 @@ func SetupNewSGNEnv(sgnParams *SGNParams, manual bool) {
 		configFileViper.Set(common.FlagEthDPoSAddress, E2eProfile.DPoSAddr.Hex())
 		configFileViper.Set(common.FlagEthLedgerAddress, E2eProfile.LedgerAddr.Hex())
 		configFileViper.Set(common.FlagEthSGNAddress, E2eProfile.SGNAddr.Hex())
+
+		if len(sgnParams.MinGasPrices) > 0 {
+			configFileViper.Set(server.FlagMinGasPrices, sgnParams.MinGasPrices)
```

Comment:
use `sgn.min_gas_price` (replace current `base_gas_price`) to make sgn.toml more organized